tests/base/test_logging.py

```diff
@@ -1,34 +1,42 @@
 import pytest
 
 import pydp as pd
-
+# Creating a Class TestLogging
 class TestLogging():
+    #Creating First testcase
     def test_logging(self):
         directory = 'tests'
         vlog_level = 3
         l = pd.logging.init_logging(directory, 'log_test', vlog_level)
+        #if both directory and vlog_level contains value the test will be pass
         assert l == True
 
     def test_logging1(self):
         directory = ''
         vlog_level = 3
         l = pd.logging.init_logging(directory, 'log_test.txt', vlog_level)
+        #location of directory must be given otherwise test will fail
         assert l == False
 
     def test_logging2(self):
         directory = 'tests'
         vlog_level = 3
         l = pd.logging.init_logging(directory, '', vlog_level)
+        #All the attributes of init_logging must be given otherwise test will fail
```

Comment:
A proper description of the tests needs to be given. 
This test fails if the output file is not mentioned.